css/style.css

```diff
@@ -0,0 +1,35 @@
+
+* {
+    font-family:Arial, Helvetica, sans-serif;
+    color: #fff;
+    background-color: #121212;
+    box-sizing: border-box;
+}
+
+nav > a{
+    text-decoration: none;
+    border: 2px solid #474747;
+    box-shadow: 0px 0px 5px #fff;
+    padding: 2px 5px;
+    margin: 3px 5px;
+    border-radius: 6px;
+}
+
+a#nf {
+    border: none;
+}
+
+.glass{
```

Comment:
It looks like prettier is not working correctly. Do you have the prettier extension installed and configured the setting of running prettier on saving a file?

Reply:
You're right, I forgot to check the box! It's working now, thanks.